sonic_platform_base/chassis_base.py

```diff
@@ -109,6 +109,31 @@ def get_reboot_cause(self):
         """
         raise NotImplementedError
 
+    def get_supervisor_slot(self):
+        """
+        Retrieves the slot of the supervisor-module of the modular chassis.
+        On the supervisor or line-card modules, it will return the slot of the
+        supervisor-module.
+
+        Returns:
+            An integer, the slot identifier of the supervisor module of the
+            modular-chassis.
+        """
+        return NotImplementedError
+
+    def get_my_slot(self):
+        """
+        Retrieves the slot of this module of the modular chassis.
+        On the supervisor, it will return the slot of the supervisor-module.
+        On the linecard, it will return the slot of the linecard module where
+        this instance of SONiC is running.
+
+        Returns:
+            An integer, the slot identifier of this module of the
+            modular-chassis.
+        """
+        return NotImplementedError
```

Comment:
Since these methods only apply to modular chassis, the docstrings should state what is expected of a fixed chassis platform.

Reply:
Fixed.